monailabel/interfaces/tasks.py

```diff
@@ -24,17 +24,12 @@
 
 
 def task_func(task, method):
-    import monailabel.utils.others.app_utils as app_utils
     cmd = [
-        "python3",
-        os.path.realpath(app_utils.__file__),
-        "-a",
+        "run_monailabel_user_app.sh",
```

Comment:
two options
1) set PATH to scripts folder (or pip install will add them bin), but howabout dev env (without pip)
2) scripts folders part of pip package and you can reference run_monailabel_user_app.sh the based on current python file path

Reply:
normally something like this.. can take care of if running in dev env vs pip mode

```
cmd_script = os.path.realpath(os.path.join(tasks.__file__, '..', 'scripts', 'run_monailabel_user_app.sh'))
cmd_script = cmd_script if os.path.exists(cmd_script) else 'run_monailabel_user_app.sh'
```

Reply:
There's a second issue here which we can discuss, as the `run_monailabel_user_app.sh` contains a `monai-label-utils` command [here](https://github.com/Project-MONAI/MONAILabel/blob/aihsani/training_venv/monailabel/scripts/run_monailabel_user_app.sh#L42).

We could perhaps add a flag that executes a "developmemt" branch of this script

Reply:
The script file (internal modules etc..) can always refer full path instead of entry point.. 

Entry points through pip are mainly for external or package users